### ISSUE TITLE
Use aria-current="page" in breadcrumb

### ISSUE THREAD

**Issue by Stephen-ONeil** [TBS-EACPD/infobase#1411]
https://css-tricks.com/5-accessibility-quick-wins-you-can-implement-today/#aa-quick-win-1-indicate-the-current-page

I think we should make use of this in the navigation breadcrumb. Not clear in the spec from what I see, but I expect screen readers will not communicate it right if it's not on a link, and we currently use a span for the last item in the bread crumb rather than a link. Probably it should be a link with this attribute, but maybe we give it some distinct styling (or remove the link styling, for consistency with the current non-link nature).
